src/games/strategy/triplea/printgenerator/CountryChart.java

```diff
@@ -92,7 +92,6 @@ protected void saveToFile(final PlayerID player, final PrintGenerationData print
     } catch (final FileNotFoundException e) {
       e.printStackTrace();
     } catch (final IOException e) {
-      // TODO Auto-generated catch block
```

Comment:
Exceptions are not adequately handled. FileNotFoundException, absolute path of the file not found is really useful to know when that happens. Removing the TODO makes it harder to spot this problem. 

The choices I see:
A. restore the TODO as is
B. update the TODO to say we need to handle the exceptions
C. Update the exception handling so we get some error messages with details about the runtime, and remove the todo comment